salt/modules/uboot.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+'''
+Interface to the U-BOOT environement variables
+
+(Parsing through fw_printenv/fw_setenv)
+
+External References
+-------------------
+| `U-BOOT http://www.denx.de/wiki/U-Boot`_
+
+'''
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+import salt.utils
+
+# Solve the Chicken and egg problem where grains need to run before any
+# of the modules are loaded and are generally available for any usage.
+import salt.modules.cmdmod
```

Comment:
That's a really good idea. I'll update the commit to also check the return code of fw_* calls.

Thank you!